IFC4x3/Properties/i/InstallationInstruction_0rXKb2EWH1Bgos1A$pv0QF/Documentation.md

```diff
@@ -0,0 +1 @@
+Detailed description on installation requirements that need to be specifically noted in accordance with specifications or standards. Installation requirements vary when the device is in different lots, such as subgrade, bridge, tunnel.
```

Comment:
We have this one somewhere else already, don't we?

Reply:
hmm, we have:

-  `AssemblyInstruction_3W5i6nbkP9NBt_M6PTB5jW`
> Instructions to describe how the system / equipment / facility is assembled.

-   `InstallationPlan_2beGsManD6zPMmBIW4LY_Q`
> Reference to external information source about installation or construction plan of the element.